src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,34 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    public Ball() {
+    }
+
+    public enum Color {
```

Comment:
Let’s  make this enum like a separate  class or you can use it like a private nested class in ColorSupplier

Reply:
Fixed it, thank you. Added class "ColorList" whith only enum